app/models/history.rb

```diff
@@ -257,6 +257,16 @@ def self.calculated_symptom_onset(patient: nil, created_by: 'Sara Alert System',
     create_history(patient, created_by, HISTORY_TYPES[:monitoring_change], comment)
   end
 
+  def self.send_close_conact_method_blank(patient: nil, created_by: 'Sara Alert System', type: 'Unknown')
+    comment = "The system was unable to send a monitoring complete message to this monitoree because their preferred contact method, #{type}, was blank."
+    create_history(patient, created_by, HISTORY_TYPES[:monitoring_complete_message_sent], comment)
+  end
+
+  def self.send_close_sms_blocked(patient: nil, created_by: 'Sara Alert System')
+    comment = 'The system was unable to send a monitoring complete message to this monitoree because SMS is blocked.'
```

Comment:
Might need to run this wording by PO but something like 'The system was unable to send a monitoring complete message to this monitoree because Recipient phone number blocked communication with Sara Alert` Would be in line with other Blocked SMS verbage. Otherwise this looks great!

Reply:
@mayerm94 @Bialogs @holmesie I'm about to post a message in the UI channel to ask if the new text for this history is acceptable

Reply:
@mayerm94 @Bialogs @holmesie got the OK from Alex in the UI channel for this wording